routes/api/v1.js

```diff
@@ -223,9 +222,9 @@ api.declare({
     "Modify client `name`, `expires`, `scopes` and",
     "`description`.",
     "",
-    "**Required scopes**, in addition the scopes listed",
-    "above, the caller must also posses the all the scopes that is given to",
-    "the client that is updated."
+    "**Required scopes**: in addition the scopes listed",
+    "above, the `scopes` parameter must satisfy the caller's scopes.  The",
+    "client's existing scopes need not satisfy the caller's scopes."
```

Comment:
note: `client.scopes <= caller.scopes` where `client` is the payload. Hence, caller must satisfy the scopes granted to the client.